.github/scripts/test_check_pr_quality.py

```diff
@@ -0,0 +1,377 @@
+import unittest
+
+from check_pr_quality import (
+    IssueType,
+    Severity,
+    Location,
+    PatternMatch,
+    QualityIssue,
+    get_context,
+    check_pr_quality,
+    generate_actions,
+    NoAction,
+    LabelPR,
+    CommentOnPR,
+    ClosePR,
+)
+
+
+class TestGetContext(unittest.TestCase):
+    """Test the get_context function"""
+
+    def test_get_context_normal_case(self):
+        text = "This is a test string for context extraction"
+        result = get_context(text, 10, 14, 5)  # "test" with 5 chars context
```

Comment:
This is a lot clearer if we use keyword arguments:

```suggestion
        result = get_context(text, match_start=10, match_end=14, context_chars=5)
```

The same applies to the rest of the tests in this class.